src/wallet/utils.rs

```diff
@@ -86,7 +90,7 @@ impl Older {
     }
 }
 
-impl<Pk: MiniscriptKey> Satisfier<Pk> for Older {
+impl<ToPkCtx: Copy, Pk: MiniscriptKey + ToPublicKey<ToPkCtx>> Satisfier<ToPkCtx, Pk> for Older {
```

Comment:
On line 63 you call this same constraint `PkCtx`. Probably best to use `ToPkCtx` in all cases, since that's what `rust-miniscript` uses.